core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/screen/ScreenCollector.java

```diff
@@ -222,6 +224,25 @@ public void setParameters(Map<String, String> params) throws ParametersException
     }
   }
 
+  private BufferedImage getSubImage(BufferedImage fullImg, Point point, Dimension size) {
+    int width = size.getWidth();
+    int height = size.getHeight();
+    if (point.getX() + width > fullImg.getWidth()) {
+      width = fullImg.getWidth() - point.getX();
+    }
+    if (point.getY() + height > fullImg.getHeight()) {
+      height = fullImg.getHeight() - point.getY();
+    }
+    if (point.getX() + width == 0) {
```

Comment:
I think that instead of those 2 `if`'s, we can change `if` above to:
` width = Math.max(1, fullImg.getWidth() - point.getX()` (same for height)

Reply:
done ;)